pkg/controller.v1/mxnet/mxjob_controller.go

```diff
@@ -123,7 +123,7 @@ type MXJobReconciler struct {
 
 func (r *MXJobReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = log.FromContext(ctx)
-	logger := r.Log.WithValues(mxjobv1.Singular, req.NamespacedName)
+	logger := r.Log.WithValues("mxjobs", req.NamespacedName)
```

Comment:
@Jeffwan Do we want to keep `Singular` and `Plural` in the constants also ?

Reply:
Yeah, nice catch. I've moved them to constants.go now